File: Pyhon/1038.py
lista = input().split(" ")
codigo = float(lista[0])
qtde = float(lista[1])

if codigo == 1:
    valor = 4.00*qtde
elif codigo == 2:
    valor = 4.50*qtde
elif codigo == 3:
    valor = 5.00*qtde
elif codigo == 4:
    valor = 2.00*qtde
elif codigo == 5:
    valor = 1.50*qtde

print("Total: R$ %.2f"%valor)

File: C/1038.c
#include <stdio.h>
int main(){

int codigo;
int quantidade;
float valor;

scanf("%d %d",&codigo,&quantidade);

switch(codigo){

case 1: valor = 4.00*quantidade;
   printf("Total: R$ %.2f\n",valor);
break;

case 2: valor = 4.50*quantidade;
   printf("Total: R$ %.2f\n",valor);
break;

case 3: valor = 5.00*quantidade;
   printf("Total: R$ %.2f\n",valor);
break;

case 4: valor = 2.00*quantidade;
   printf("Total: R$ %.2f\n",valor);
break;

case 5: valor = 1.50*quantidade;
   printf("Total: R$ %.2f\n",valor);
break;

return 0;
}

}
